forc-client-ops/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "forc-client-ops"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+anyhow = "1"
+clap = { version = "3", features = ["derive"] }
+forc = {path = "../forc"}
```

Comment:
Ahh I missed this. Can you share why we need the `forc` crate itself? Is it just for the git tag?

In retrospect, that constant might be better off living in the `forc-pkg` crate. This is because the git tag constant is solely used with `forc-pkg` functionality (i.e. the inclusion of implicit std). This change might be better off made in a dedicated PR though. This would also help both `sway-lsp` and the recent SDK/Sway testing as they would no longer require depending on `forc` itself, and would only require `forc-pkg` (and as a result, not need the entire `clap` dependency tree).

If this is the case, would you mind if we open a dedicated PR for this first?

Reply:
Just a heads up, I'm going to have a go at moving this constant now as I think it's the right move either way!

Reply:
Hmm, we need `forc` because before running `deploy` or `run` we need to build. We are currently doing it by creating a `BuildCommand` (which lives in `forc`) from the parameters we pass to `deploy` or `run`. But I can probably refactor that place to only use `forc-pkg`. I will have a look at this shortly 👍🏻

Reply:
I took a second look, and it seems like we can remove `forc` dependency but there is some logic around `build` in forc regarding `build_profile` and some other stuff. To remove that we may need to duplicate that logic on the `forc-client` side. Would that be acceptable?

Reply:
If we can use `forc-pkg` directly I think that would be best. If there is duplicated code, perhaps there's a chance to refactor some of it into `forc-pkg`?

Ideally we shouldn't need `forc` directly, as I think `forc` itself should just be a thin CLI wrapper on top of `forc-pkg` + plugin support.

Reply:
That makes perfect sense. I opened up #2478 for that refactoring and marking this PR as draft as this one is blocked until we land that one 👍